build.sbt

```diff
@@ -89,6 +99,7 @@ lazy val doc = project
     tutTargetDirectory := (baseDirectory.value).getParentFile,
     libraryDependencies ++= Seq("io.circe" %% "circe-generic" % circeVersion)
   )
+  .settings(publishSettings)
```

Comment:
The top level project uses the "aggregate" function (line 45) so I don't think the settings are needed on each project individually? I may be wrong.